modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/objectLayoutContext.tsx

```diff
@@ -361,6 +369,19 @@ const layoutReducer = (state: TState, action: TAction) => {
 
 			return newState;
 		}
+		case TYPES.DELETE_OBJECT_LAYOUT_BOX_CATEGORIZATION: {
+			const {boxIndex, tabIndex} = action.payload;
+
+			const newState = {...state};
+
+			// Delete object layout box
```

Comment:
Hey @HenriquePereiraa, since this function is very small and the `type` name is very clear, I don't think we need that comment, you can remove it.